spec/features/04_user_edits_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+feature "user edits their profile page" do
+  let!(:user_1) { User.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.